src/components/CVModal/container.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { fetchCV } from 'Services';
+import { getEmployeeCV } from 'Reducers/selected-options';
+
+const CVModalContainer = Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      employeeCV: PT.shape({ id: PT.number, label: PT.string }).isRequired,
+    };
+
+    state = { cv: null };
```

Comment:
What data does CV contain?

Reply:
the URL pointing to the CV. I have changed the var name to cvURL